.github/workflows/eco-vcenter-ci.yaml

```diff
@@ -0,0 +1,54 @@
+---
+name: Ansible Eco vCenter Integration Test
+on:
+  pull_request:
+    branches:
+      - '**'
+  push:
+    branches:
+      - main
+      - 'release-\d.\d\d'
+permissions:
+  contents: read
+jobs:
+  ansible_integration_test:
+    runs-on: ["self-hosted", linux, X64]
+    steps:
+      - name: Update pip, git
+        if: runner.os == 'Linux' && startsWith(runner.name, 'ubuntu')
+        run: |
+          set -euxo pipefail
+          sudo apt-get update
+          sudo apt-get install podman
+
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          path: ansible_collections/cloud/vmware_ops
+
+      - name: Debug Secrets
```

Comment:
I dont think we should print the password out :)

Reply:
opss, remove it, tnx!